src/Components/ArticleCard.jsx

```diff
@@ -1,46 +1,69 @@
 import { ArrowBigUp, ArrowBigDown, Share, MessageCircle } from 'lucide-react';
+import { Link, useNavigate } from 'react-router-dom';
+import { formatDate } from "../../utils/formatting";
 
+const ArticleCard = ({ article }) => {
+  const navigate = useNavigate();
 
+  const handleCardClick = (e) => {
+    if (e.target.tagName !== 'A' && e.target.tagName !== 'BUTTON') {
+      navigate(`/articles/${article.article_id}`);
+    }
```

Comment:
I would expect the functionality of this app to still work without useNavigate here. I think we can remove line 6 and handleCardClick function and our app will still route to a single article